File: app/src/main/java/com/unicorn/sxmobileoa/login/parse/Response.java
package com.unicorn.sxmobileoa.login.parse;

public class Response {

    private String code;
    private String msg;
    private String randCode;
    private String randCodeSec;
    private String loginBusiType;
    private String time;
    private String seqM;
    private String secM;
    private String seqD;
    private String seqR;
    private Parameters parameters;

    //

    public Parameters getParameters() {
        return parameters;
    }

    public void setParameters(Parameters parameters) {
        this.parameters = parameters;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public String getRandCode() {
        return randCode;
    }

    public void setRandCode(String randCode) {
        this.randCode = randCode;
    }

    public String getRandCodeSec() {
        return randCodeSec;
    }

    public void setRandCodeSec(String randCodeSec) {
        this.randCodeSec = randCodeSec;
    }

    public String getLoginBusiType() {
        return loginBusiType;
    }

    public void setLoginBusiType(String loginBusiType) {
        this.loginBusiType = loginBusiType;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getSeqM() {
        return seqM;
    }

    public void setSeqM(String seqM) {
        this.seqM = seqM;
    }

    public String getSecM() {
        return secM;
    }

    public void setSecM(String secM) {
        this.secM = secM;
    }

    public String getSeqD() {
        return seqD;
    }

    public void setSeqD(String seqD) {
        this.seqD = seqD;
    }

    public String getSeqR() {
        return seqR;
    }

    public void setSeqR(String seqR) {
        this.seqR = seqR;
    }
}

File: app/src/main/java/com/unicorn/sxmobileoa/app/chongqing/Response.kt
package com.unicorn.sxmobileoa.app.chongqing

import com.google.gson.internal.LinkedTreeMap

data class Response(
        var code: String,
        var msg: String,
        var busiCode: String,
        var randCode: String,
        var time: String,
        var seqM: String,
        var secM: String,
        var seqD: String,
        var seqR: String,
        var thirdFlow: String,
        var parameters: LinkedTreeMap<String, String>? = null
) {

    /**
     * code : 000000
     * msg : 请求成功
     * busiCode : getFyList
     * randCode : 92cfb7f8-5c05-42d5-9247-36a67829d5a3
     * time : 20170622141844
     * seqM :
     * secM :
     * seqD :
     * seqR :
     * thirdFlow : 1234567
     */

}

